superset/sql/parse.py

```diff
@@ -125,6 +125,37 @@ def __eq__(self, other: Any) -> bool:
         return str(self) == str(other)
 
 
+@dataclass(eq=True, frozen=True)
+class Partition:
```

Comment:
this module is getting refactored as we speak by @betodealmeida , he can probably help making sure this lands safely